src/dotnet/commands/dotnet-add/dotnet-add-package/LocalizableStrings.resx

```diff
@@ -165,4 +165,7 @@
   <data name="CmdDGFileIOException" xml:space="preserve">
     <value>Unable to generate a temporary file for project '{0}'. Cannot add package reference. Please clear the temp directory and try again.</value>
   </data>
+  <data name="CmdInteractiveRestoreDescription" xml:space="preserve">
+    <value>Allows the command to stop and wait for user input or action (for example to complete authentication)</value>
```

Comment:
Nit: for conformity with our other option descriptions, could you add a period on the end of this string?